MACServerExploit.py

```diff
@@ -111,7 +114,13 @@ def __send_end__(self):
 
     def __send_init__(self):
         self.sock.sendto(self.__build_packet__(self.START), self.ADDR)
+        n = datetime.datetime.now()
         while not self.connected:
+            if (datetime.datetime.now()-n).total_seconds() > 3:
+                #self.close() # This does not seem to work as expected.
+                self.work = False
+                #self.sock.setblocking(0) # This doesn't seem to affect the "recvfrom" if it is already waiting.
```

Comment:
I made a few attempts here to abort the `__recv_manager__`'s `recvfrom`, but nothing seemed to really work.